### DISCUSSION TITLE
49-2, 50-2 문제 해석요청합니다.

### DISCUSSION THREAD

**Discussion by Mang-Joo** [I-JUNNYLAND-I/algorithm#242] [Q&A] [answered]
49-2 [Q2](https://leetcode.com/problems/fair-candy-swap/)
50-2 [Q2](https://leetcode.com/problems/special-array-with-x-elements-greater-than-or-equal-x/)
아니 미디엄이 더 쉬움

**Comment by wlsdn93** [comment]
[Day49 Q2 : Fair Candy Swap](https://leetcode.com/problems/fair-candy-swap/)

앨리스와 밥은 서로 다른 개수의 사탕을 갖고 있습니다. 
`ailiceSizes[i]` 와 `bobSizes[i]`는 `i`번째 상자에 들어있는 사탕 개수를 의미합니다.
앨리스와 밥은 서로 친구이기 때문에, 사탕 박스를 교환하여 둘이 같은 수의 사탕을 갖도록 하고자 합니다. 
이 때, 길이가 2인 정수배열 `answer` 을 반환해주세요. `answer[0]`은 앨리스가 교환해야 하는 박스에 들어있는 사탕의 개수를, `answer[1]`은 밥이 교환해야하는 박스에 들어있는 사탕의 개수를 의미합니다. 

Example
```
Input: aliceSizes = [1,2], bobSizes = [2,3]
앨리스는 0번 인덱스 상자를, 밥은 0번 인데스 상자를 서로 교환해야 총 개수가 4개로 같아집니다.
따라서 정답은 [1, 2] 입니다. 
```

**Comment by wlsdn93** [comment, marked as answer]
[Day50 Q2 : Special Array With X Elements Greater Than or Equal X](https://leetcode.com/problems/special-array-with-x-elements-greater-than-or-equal-x/)

음수가 아닌 정수배열 `nums` 가 주어질 때, 임의의 정수 `x`에 대하여 만약 `nums` 안에 `x`보다 크거나 같은 숫자의 개수가 `x`인 `x`가 존재한다면, `nums`는 **특별**하다고 할 수 있습니다.
만약 `nums`가 특별하다면 `x`를 반환하고, 아닌경우 `-1`를 반환해주세요.  
만약 `nums`가 특별하다면, `x`값은 단 하나만 존재합니다. 

Example
```
Input: nums = [3,5]
nums는 2보다 크거나 같은 수를 2개 포함하고 있으므로, 정답은 2 입니다.
```
